app/src/lang/translations/en-US.yaml

```
@@ -347,6 +347,7 @@ clear_items: Clear Items
 reset_to_default: Reset to Default
 undo_changes: Undo Changes
 notifications: Notifications
+mark_all_as_read: Archive All
```

I do think `read` and `archive` are different enough in meaning that we should change the key. Agree?

```suggestion
archive_all: Archive All
```

Maybe? Or something like

```suggestion
archive_all_items: Archive All
```